packages/logging/src/redux/README.md

```diff
@@ -0,0 +1,32 @@
+# Redux error logger middleware
+
+This module register a Redux middleware that sends a log to an endpoint.
+
+You need to configure the logger, depending on the framework you use
+* angularjs logger [documentation](../angular/README.md)
+* react logger [documentation](../react/README.md)
+
+## Configuration
+
+You need to configure the logger in order to fit your backend logging service and add your app state.
+
+```javascript
+import { createStore, applyMiddleware } from 'redux';
+import initReactLogger from '@talend/log/lib/react';
+import tLoggerMiddleware from '@talend/log/lib/redux';
```

Comment:
why *t*LoggerMiddleware ?

Reply:
`t` for Talend. But it's a default import you can name it whatever you want, it's just an example